pkg/apis/contrail/v1alpha1/tests/config_vrouter_test.go

```diff
@@ -162,26 +162,83 @@ func TestVrouterCustomEnvVariablesConfigMap(t *testing.T) {
 		Namespace: "default",
 	}
 
-	environment := SetupEnv()
-	cl := *environment.client
+	t.Run("With non-empty map EnvVariablesConfig", func(t *testing.T) {
+		environment := SetupEnv()
+		cl := *environment.client
 
-	environment.vrouterResource.Spec.ServiceConfiguration.VrouterEncryption = true
-	environment.vrouterResource.Spec.ServiceConfiguration.PhysicalInterface = "eth0"
+		if err := environment.vrouterResource.InstanceConfiguration(request,
+			&environment.vrouterPodList, cl); err != nil {
+			t.Fatalf("get configmap: (%v)", err)
+		}
+		if err := cl.Get(context.TODO(), configMapNamespacedName, &environment.vrouterConfigMap2); err != nil {
+			t.Fatalf("get configmap: (%v)", err)
+		}
```

Comment:
I do not think this is required, because only after these lines you are adding the custom env variables and run InstanceConfiguration once again.